esrally/racecontrol.py

```diff
@@ -287,13 +287,13 @@ def race(cfg, sources=False, build=False, distribution=False, external=False, do
 
 
 def set_default_hosts(cfg, host="127.0.0.1", port=9200):
-    configured_hosts = cfg.opts("client", "hosts", mandatory=False)
-    if configured_hosts:
-        logger.info("Using configured hosts %s" % configured_hosts)
+    configured_hosts = cfg.opts("client", "hosts")
+    if len(configured_hosts()) !=0 :
```

Comment:
I think it would be better to use an explicit property name here instead of `()`?